fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraNodesIT.java

```diff
@@ -44,7 +44,8 @@
  * @author ajs6f
  */
 @Ignore //TODO Fix these tests
-public class FedoraNodesIT extends AbstractResourceIT {
+public class
```

Comment:
Non-standard choice of formatting. Can updates to this class be reverted?

Reply:
I'll revert - not sure how this happened.